src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,50 @@
+.contacts {
+
+  &__block {
+    @include onTablet {
+      display: flex;
+      gap: 75px;
+    }
+  }
+
+  &__data {
+    @include onDesktop {
+      display: flex;
+      gap: 100px;
+    }
+  }
+
+  &__part {
+    margin-top: 48px;
+
+    @include onTablet {
+      display: block;
+      flex-direction: column;
+    }
+  }
+
+  &__subtitle-1 {
+    font-size: 16px;
+    line-height: 27px;
+    color: $colorBlack;
+    text-decoration: none;
+  }
+
+  &__subtitle-2 {
+    margin-top: 16px;
+    @include smallTextProperties;
+  }
+
+  &__subtitle-3 {
+    margin-top: 8px;
+    @include smallTextProperties;
+  }
+
+  &__info {
+    margin-top: 4px;
+    color: $colorBlack;
+    @include textProperties;
+    text-decoration: none;
```

Comment:
duplicated value, you already have this prop from link

Reply:
fixed